_config.yml

```diff
@@ -1,4 +1,4 @@
-name: Liblouis*
+name: Liblouis
```

Comment:
The reason for the `*` was/is the fact that the website is supposed to be for both liblouis and liblouisutdml. But I guess the `*` might be confusing also, so maybe it's ok to drop it

Reply:
Yes, I know. But I don't think anyone else knows, it's not clear enough. People also just consider Liblouisutdml as part of the Liblouis software suite. For most people it's just one.

Reply:
Yes, you're probably right